app/components/ilios-event.js

```diff
@@ -127,49 +127,50 @@ export default Component.extend({
       promise: defer.promise
     });
   }),
-  courseLearningMaterials: computed(
-    'i18n.locale',
-    'offering.session.course.learningMaterials.@each.learningMaterial.[title,absoluteFileUri]',
-    function(){
-      let defer = RSVP.defer();
+  courseLearningMaterials: computed('i18n.locale', function() {
```

Comment:
yes, i did that originally. however, this will cause the same XHR for retrieving course LMs multiple times.
since its a computed function, I suspect this method is firing off every time `thesession.course` changes. 
any idea as how to overcome this?